### DISCUSSION TITLE
A Gentoo package for this application

### DISCUSSION THREAD

**Discussion by Leo3418** [mjakeman/extension-manager#25] [Show and tell]
As a Gentoo GNOME user, I have created some ebuilds for this application and added them to GURU.  They can be found at:

<https://github.com/gentoo/guru/tree/master/gnome-extra/extension-manager>

If you happen to need them, just [enable GURU](https://wiki.gentoo.org/wiki/Project:GURU/Information_for_End_Users) on your system, sync it, and emerge `gnome-extra/extension-manager`.  Note that at this point, some dependencies of this application are masked in the Gentoo repository, so you need to unmask them as well as this package.

**Comment by mjakeman** [comment]
This is really cool. Thanks for sharing!

**Comment by Leo3418** [comment]
Update: The package has been moved to GURU, so it's now even easier to install it on Gentoo! It should show up in the Repology badge in README the next time the crawler runs. I've updated the URLs in the OP accordingly.

**Reply by mjakeman** [reply]
Awesome work!

In my effort to be more package maintainer friendly, I've created https://github.com/mjakeman/extension-manager/issues/94. You may want to subscribe to that to be pinged when a new release is published :)
